packages/db/src/translate/granules.ts

```diff
@@ -179,7 +179,7 @@ export const translateApiGranuleToPostgresGranule = async (
     updated_at: new Date(dynamoRecord.updatedAt),
   };
 
-  return granuleRecord;
+  return <PostgresGranule>removeNilProperties(granuleRecord);
```

Comment:
This is the big change.

Reply:
Still thinking through the review conversation/comment re: other uses, but I believe I agree with your assessment.